packages/api/src/controllers/access-control.ts

```diff
@@ -21,7 +21,7 @@ app.use("/access-control", accessControl);
 
 accessControl.post(
   "/gate",
-  authorizer({ anyAdmin: true }),
+  authorizer({}),
```

Comment:
You should remove this middleware as the API should be public. There is no authorization.

Reply:
Didn't get we wanted it full public, still thought with a token required. Sounds good and removed

Reply:
Cool!